examples/terraform-azure-aks-example/README.md

```diff
@@ -0,0 +1,61 @@
+# Terraform Azure AKS Example
+
+This folder contains a simple Terraform moudle that deploys a simple AKS cluster in [Azure](https://azure.microsoft.com/) to demonstrate how you can use Terratest to write automated tests for your Azure Terraform code. 
```

Comment:
```suggestion
This folder contains a Terraform module that deploys a basic AKS cluster in [Azure](https://azure.microsoft.com/) to demonstrate how you can use Terratest to write automated tests for your Azure Terraform code. 
```

We try to avoid the word "simple" because it can be intimidating if you struggle to understand it, and even the most basic examples can be complex to understand if you've never used Azure.